plugins/populator/frontend/app.js

```diff
@@ -5,8 +5,13 @@ var exported = {},
 
 (function(plugin) {
     plugin.init = function(app) {
-        app.get(countlyConfig.path + '/:id/demo-page.html', function(req, res) {
-            var pageSourceCode = fs.readFileSync(path.resolve(__dirname, 'public/templates/demo-page.html'));
+        const predefinedTypes = ["demo-banking.html", "demo-gaming.html", "demo-navigation.html", "demo-healthcare.html", "demo-ecommerce.html"];
+        app.get(countlyConfig.path + '/:id/:page', function(req, res) {
```

Comment:
I think this path pattern breaks tests
that is why Travis cannot pass the tests.
I think we cannot use any substring pattern there, but there must be some unique prefix, like 
```
app.get(countlyConfig.path + '/populator/:id/:page', function(req, res) {
```
Also, can you recheck the warnings on LGTM about escaping?